src/sdp/media/ersip_sdp_attr_rtpmap.erl

```diff
@@ -0,0 +1,209 @@
+%%
+%% Copyright (c) 2020 Dmitry Poroh
+%% All rights reserved.
+%% Distributed under the terms of the MIT License. See the LICENSE file.
+%%
+%% SDP media attribute rtpmap 
+%%
+
+-module(ersip_sdp_attr_rtpmap).
+
+-export([new/3, new/4]).
+
+-export([payload_type/1]).
+-export([encoding_name/1]).
+-export([clock_rate/1]).
+-export([encoding_params/1]). 
+-export([set_payload_type/2]).
+-export([set_encoding_name/2]).
+-export([set_clock_rate/2]).
+-export([set_encoding_params/2]).
+
+-export([parse/1]).
+-export([assemble/1]).
+-export([assemble_bin/1]).
+
+-define(IS_DIGIT(X),    (X >= $0 andalso X =< $9)).
```

Comment:
This is already defined in:
https://github.com/poroh/ersip/blob/master/include/ersip_abnf.hrl